### PR TITLE
Add a note to OpenCL

### LINKED ISSUE #2842: [feature] OpenCL support
Original commit: https://github.com/qgis/QGIS/commit/0b502ff5b9b6925c694ae1709918e14233daf6cc by web-flow

[feature] OpenCL support

### PR DESCRIPTION
**Pull request by ghtmtt**
Goal: add a small note to OpenCL support. I think #2842 is already documented but maybe this note can be a little bit more helpful

Ticket(s): fix #2842

- [x] Backport to LTR documentation is required

### CODE REVIEW

File: docs/user_manual/introduction/qgis_configuration.rst [hunk 1]
```diff
@@ -1009,6 +1009,9 @@ OpenCL acceleration settings.
 
    Acceleration tab
 
+.. note:: depending on your machine you have to install additional libraries to
+  enable the OpenCL acceleration.
```
Comment (havatv) [inline docs/user_manual/introduction/qgis_configuration.rst:1013]:
```suggestion
.. note:: depending on your hardware and software, you may have to install additional
   libraries to enable OpenCL acceleration.
```

**Review by havatv — APPROVED** [review]
Looks fine. I made a trivial changed, and committed to save us all some work.

**Comment by DelazJ** [comment]
> I think #2842 is already documented

You sure? The only reference I can find is https://docs.qgis.org/testing/en/docs/user_manual/introduction/qgis_configuration.html#acceleration-settings (not really documenting a setting, right?). Or is there another place?
Anyway, not sure the single text in the section has to be within a `.. note::

**Comment by ghtmtt** [comment]
> You sure? The only reference I can find is https://docs.qgis.org/testing/en/docs/user_manual/introduction/qgis_configuration.html#acceleration-settings (not really documenting a setting, right?). Or is there another place?

pretty sure yes :blush: I think we don't need to explain what OpenCL is, we need that users know that they can enable it.

I couldn't find any other place in our manual

> Anyway, not sure the single text in the section has to be within a `.. note::

do you suggest a *normal* sentence. What I thought is that the manual already has the OpenCL tab and we have to inform the user that is not automatic that OpenCL is enabled, maybe (s)he has to install additional software

**Comment by DelazJ** [comment]
> do you suggest a normal sentence. 

Yes. To me, a note brings details you don't have in the text. And if no text, let's make the note the text 😄

**Comment by ghtmtt** [comment]
@DelazJ sorry for the late commit.. I removed the note, hoping it is clear and that we can merge it :)

**Pull request merged**